web/react/components/user_settings/user_settings_general.jsx

```diff
@@ -209,8 +234,11 @@ export default class UserSettingsGeneralTab extends React.Component {
     setupInitialState(props) {
         var user = props.user;
         var emailEnabled = global.window.config.SendEmailNotifications === 'true';
+        var emailVerificationEnabled = global.window.config.RequireEmailVerification === 'true';
+
         return {username: user.username, firstName: user.first_name, lastName: user.last_name, nickname: user.nickname,
-                        email: user.email, picture: null, loadingPicture: false, emailEnabled: emailEnabled};
+                        email: user.email, confirmEmail: '', picture: null, loadingPicture: false, emailEnabled: emailEnabled,
+                        emailVerificationEnabled: emailVerificationEnabled, emailChangeInProgress: false};
```

Comment:
Mainly because the `emailEnabled` field was there as well (which I probably put there).  Good point though I'll move it out.